assets/scripts/actions.js

```diff
@@ -7,6 +7,12 @@ import store from './store.js';
 import makeBuildStatus from "./buildStatus.js";
 import { handleErrors, logMessage, delay } from "./utils";
 
+const logout = () => {
+  store.mutations.setLogin(undefined)
+  store.mutations.invalidateToken()
+  store.mutations.removeSite()
+}
```

Comment:
```suggestion
  page("/login")
}
```

Il risque d'y avoir des petits bugs un peu complexes à déclencher (il faut que la personne retourne sur une route qui utilise le token) du fait que le token est toujours dans l'instance `databaseAPI`. Mais c'est tellement improbable que ça me va comme ça pour le moment